region/brinstar/green.json

```diff
@@ -1110,11 +1109,10 @@
               "requires": [
                 {"or": [
                   "Morph",
-                  "FIXME"
+                  "canTunnelCrawl"
```

Comment:
I think this needs to be a separate, `canMorphlessTunnelCrawl` tech. Because without morph, you are going to softlock if you mess up.

Also, the same tech is required to make it back from 4 to 1.

Finally, there's a one-way strat to go from 1 to 4 with `canTurnaroundAimCancel`, it just won't get you back out.

Reply:
Changing

Reply:
I guess this means I should probably add Morph to `canTunnelCrawl`'s reqs though, and update the note.